Enterprise-CMCS/macpro-mdct-mfp
# Refactor Entity Status Icon

### Description

Refactor the EntityStatusIcon component to
- Simplify logic
- Remove duplicate styles
- Unify icons in all views
- Make the status types fit in the enum

Notes for review: I recommend you review the commits one-by-one

### Related ticket(s)


CMDCT-3690

---
### How to test

- Go to https://d11w0gb4ih2yvu.cloudfront.net/
- Log in as `cypressstateuser@test.com`
- View the initiative sections for WP and SAR, in both report and PDF views
  - Verify the icons are consistent
- Modify one of the initiatives in the SAR
  - Verify the statusing works as expected

No changes to display or logic (other than using lighter version of checks and exes for all locations)
---
### Pre-review checklist
